### PR TITLE
[NetCore] Add a new medium post in NetCore

### PR DESCRIPTION
**Pull request by gussiciliano**
### SUMMARY
Add link, Migrating from .Net Core 1 to .Net Core 2.0

### CODE REVIEW

File: net-core/README.md [hunk 1]
```diff
@@ -17,3 +17,4 @@ At [Wolox](http://wolox.com.ar), we have developed the [NET Core kickoff guide](
 
 - [Creating your first .NET Core project on Linux](https://medium.com/wolox-driving-innovation/how-to-create-your-first-net-e2223dedb74f)
 - [How to create a .NET Core project on Linux using MVC and Postgres](https://medium.com/wolox-driving-innovation/create-an-mvc-web-app-in-net-core-30f92cf94a75)
+- [Migrating from .NET Core 1 to .Net Core 2.0 ](https://medium.com/wolox-driving-innovation/)
```
Comment (dtilve) [inline net-core/README.md]: Could it be the actual post is missing? This link redirects to the main Wolox medium page.

Comment (gussiciliano) [reply net-core/README.md]: Yes, I am going to post in medium in the next week, but I needed to do the PR :P

Comment (dtilve) [reply net-core/README.md]: LOL just read the [WIP]

**Comment by mvbattan** [comment]
@gussiciliano Please follow the PR naming convention described in https://github.com/Wolox/tech-guides/pull/90
Thank you ! 😊

File: net-core/README.md (hunk 1 above)
Comment (gussiciliano) [reply net-core/README.md]: Jaja oks :P

**Comment by gussiciliano** [comment]
@dtilve sorry for the late with this PR, it is ready now :D

**Pull request merged**